### DISCUSSION TITLE
posts/notice-understand-info-asymmetry/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#20] [Announcements]
# posts/notice-understand-info-asymmetry/

信息差是我们生活中常见的现象，它不仅影响我们的学习和工作，还可能被人利用来欺骗我们。通过认识、理解和重视信息差，我们可以更好地应对生活中的各种挑战，提升自己的能力和警惕性。

https://blog.coldwind.top/posts/notice-understand-info-asymmetry/

**Comment by Three3tree** [comment]
B站过来的，感谢大佬分享的知识，不局限于编程方面，还有一些观念和方法，对我工作学习帮助很大。

**Comment by dongpo1996** [comment]
很认同,真的需要走出自己的舒适区,这样才会有思考和进步.

**Comment by gniknyh** [comment]
感激大佬在做的分享知识来减少信息差的行为.
